src/components/action-sheet/action-sheet.ios.scss

```diff
@@ -103,13 +106,25 @@ $action-sheet-ios-button-cancel-font-weight:            600 !default;
   @include border-radius($action-sheet-ios-border-radius);
   @include margin(null, null, $action-sheet-ios-group-margin-bottom - 2, null);
 
-  overflow: hidden;
+  overflow: scroll;
```

Comment:
Should we add overflow-y only? So borders do not show on overflow

Reply:
+ Maybe move this rule to the common `action-sheet.scss` file?